frontend/awx/analytics/Reports/AutomationCalculator.tsx

```diff
@@ -87,38 +87,41 @@ export default function AutomationCalculator(props: { schema: ChartSchemaElement
 
   const [specificError, setSpecificError] = useState<string>('');
 
-  const defaultParams: ParamsType = {
-    status: ['successful'],
-    org_id: [],
-    cluster_id: [],
-    template_id: [],
-    inventory_id: [],
-    quick_date_range: 'roi_last_year',
-    job_type: ['job'],
-    sort_options: 'successful_hosts_savings',
-    sort_order: 'desc',
-    start_date: undefined,
-    end_date: undefined,
-    limit: '6',
-    offset: '0',
-    only_root_workflows_and_standalone_jobs: true,
-    template_weigh_in: undefined,
-    attributes: [
-      'elapsed',
-      'host_count',
-      'total_count',
-      'total_org_count',
-      'total_cluster_count',
-      'successful_hosts_total',
-      'successful_elapsed_total',
-    ],
-    group_by: 'template',
-    group_by_time: false,
-  };
+  const defaultParams: ParamsType = useMemo(
```

Comment:
Why is `useMemo` used here? This is static with no expensive function.

Reply:
defaultParams is used inside
```
  const getParams = useCallback(() => {
```

If defaultParams is not using useMemo then we get an error
```
The 'defaultParams' object makes the dependencies of useCallback Hook (at line 138) change on every render. Move it inside the useCallback callback. Alternatively, wrap the initialization of 'defaultParams' in its own useMemo() Hook.eslint[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)
```